packages/bugsnag_flutter_performance/lib/src/client.dart

```diff
@@ -130,6 +131,7 @@ class BugsnagPerformanceClientImpl implements BugsnagPerformanceClient {
         networkRequestCallback,
     String? releaseStage,
     List<String>? enabledReleaseStages,
+    List<String>? tracePropagationUrls,
```

Comment:
Given that these are matches as regex's I think this list should make that clearer:
```suggestion
    List<RegExp>? tracePropagationUrls,
```